src/lib/dev.ts

```diff
@@ -36,13 +50,13 @@ function createIndexRenderer(appRoot, jsonModules, onInit, onUpdate, onBeforeIni
         }
         return String(a.data.meta.id || a.page).localeCompare(String(b.data.meta.id || b.page));
       })
-      .map(({ page, data }) => ({
+      .map<Page>(({ page, data }) => ({
```

Comment:
Should this `Page` type be defined earlier in the flow? Like on the `const item`?